tests/ProductsTest.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace ready2order\Tests;
+namespace Tests;
```

Comment:
@krsriq please run `composer install` again as in `composer.json` the namespace was changed to `ready2order/Tests` to not have any namespace collisions.
```
"autoload": {
        "psr-4": {
            "ready2order\\": "src",
            "ready2order\\Tests\\": "tests"
        }
    },
```